python/dgl/dataloading/dataloader.py

```diff
@@ -133,6 +132,50 @@ def _get_id_tensor_from_mapping(indices, device, keys):
     return id_tensor
 
 
+def _split_to_local_id_tensor_from_mapping(indices, keys, bounds):
```

Comment:
Could we make `bounds` two arguments instead?  Like `lower_bound` and `upper_bound`?  I think they are two scalars?  `bounds[1]` and `bounds[0]` are less readable.